packages/trackerless-network/protos/NetworkRpc.proto

```diff
@@ -29,12 +29,12 @@ service TemporaryConnectionRpc {
 }
 
 message MessageRef {
```

Comment:
And entity which has these fields is called `MessageID` in our current protocol. And `MessageRef` is a simpler entity which contains just `timestamp` + `number` (usually a pointer to previous message of some message, i.e. we already know the other fields which are equal to that message).

Is in intentional that we have only one type? Maybe we'd need both type? (If we choose use only one type, we should maybe rename this to `MessageID`?)